indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -228,10 +234,24 @@ protected void innerMap(
         throw new ISE("WTF?! No bucket found for row: %s", inputRow);
       }
 
+      final long truncatedTimestamp = granularitySpec.getQueryGranularity().truncate(inputRow.getTimestampFromEpoch());
+      final byte[] hashedDimensions = hashFunction.hashBytes(
+          HadoopDruidIndexerConfig.jsonMapper.writeValueAsBytes(
+              Rows.toGroupKey(
+                  truncatedTimestamp,
```

Comment:
Probably doesn't matter, but I don't think the truncatedTimestamp is technically needed in the hashed dimensions set.